languages/swift/exercises/concept/tuples/Sources/tuples/tuples.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+func cartesianToPolar(_ cart: (x: Double, y: Double)) -> (r: Double, theta: Double) {
+  fatalError("Please implement the cartesianToPolar(_:) function.")
+}
+
+func combineRecords(
+  production: (toy: String, id: Int, prouctLead: String),
```

Comment:
Strings are used here, but I don't see them as a prerequisite in the `config.json` file